Firmware/OpenSteamController/src/eeprom_access.c

```diff
@@ -158,8 +158,9 @@ int eepromCmdFnc(int argc, const char* argv[]) {
 		uint32_t num_words = strtol(argv[4], NULL, 0);
 		retval = eepromReadToConsole(word_size, addr, num_words);
 	} else if (!strcmp("write", argv[1])) {
-		//TODO: imeplement this
-		printf("eeprom writing not implemented yet\n");
+		printf("experimental eeprom write\n");
+		uint32_t arr[1] = { strtol(argv[4], NULL, 16) };
+		eepromWrite(addr, arr, 1);
```

Comment:
Since writing to EEPROM could brick a user's controller I'd like to see another user prompt before writing the value (e.g. "You are about to write value 0x%08x to address 0x%08x in EEPROM. This could brick your controller. Are you sure you want to do this [Y/n]?")